doc/rm/comportability_specification/index.md

```diff
@@ -486,6 +486,10 @@ In terms of naming convention, alerts shall be given a meaningful name that is i
 Recoverable alerts must be prefixed with `recov_*`, whereas fatal alerts must be prefixed with `fatal_*`.
 For instance, an uncorrectable parity error in SRAM could be named `fatal_parity_error`.
 
+In cases where many diverse alert sources are bundled into one alert event (see [Alert Hardware Implementation]({{< relref "#alert-hardware-implementation" >}})), it may sometimes be difficult to assign the alert event a meaningful and descriptive name.
+In such cases, it is permissible to default the alert names to just `recov` and/or `fatal`.
+Note that this implies that the peripheral does not expose more than one alert per type.
```

Comment:
Maybe "... more than one alert for that type"? I don't think we want to disallow e.g. 3 alerts: recov_foo, recov_bar and fatal.

Reply:
good point, I'll reword as you suggested.